argocd/minio/values.yaml

```diff
@@ -0,0 +1,12 @@
+minio:
+  accessKey: ZQQhDzrKcSB2eYntvQEcfE5m8xgBwdMd
+  secretKey: FqYB6mV5YtwzVs9JNTNjsSEmmHY7XsXZ
+  metrics:
+    serviceMonitor:
+      enabled: true
+  buckets:
+    - name: thanos
+      policy: none
+      purge: false
+  persistence:
+    enabled: false
```

Comment:
How is you editor configured to not add new line at end of file?

Reply:
@mcanevet 
C'est le config de Visual Studio de base. Par contre le pipeline ne passe pas, il abandonne en cours de route (timeout) visiblement le make debug retourne pas mal de valeur à vide